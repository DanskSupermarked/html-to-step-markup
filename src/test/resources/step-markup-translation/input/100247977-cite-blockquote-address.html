<p>Torsdag den 1. september inviterer <strong>føtex for de små</strong> til foredrag med den anerkendte sundhedsplejerske Helen Lyng Hansen i føtex Mariagervej Randers. Helen er blandt andet kendt fra hjemmesiden netsundhedsplejerske.dk og som forfatter til blandt andet bestselleren <u><cite>Helens bog om børn og søvn &ndash; sådan får du dit barn til at sove</cite></u>. Om børn og deres søvn siger Helen:</p> <blockquote>&rdquo;Kunsten er at forstå dit barns søvnmønster, udviklingstrin og behov for tryghed. Når du forstår det, ved du også, hvorfor det ikke hj&aelig;lper bare at slukke lyset, lukke døren og gå.&rdquo;</blockquote> <p>Helen giver dig en viden om børns søvn og udvikling, der gør det lettere for dig at forstå og tackle de søvnproblemer, I oplever derhjemme. Du får hj&aelig;lp til at håndtere helt konkrete problemstillinger, og der er rig mulighed for at stille spørgsmål.</p> <p>Alle deltagere får udleveret en <b>føtex for de små</b> goodie bag med fine sager til både mor og den lille samt et gavekort til føtex på 50,- Der er begr&aelig;nset antal pladser, så tøv ikke for l&aelig;nge med at bestille.</p> <p>Tidspunkt: Torsdag 1. september kl. 18-20</p> <address>føtex Mariagervej Randers<br /> Mariagervej 175<br /> 8920 Randers NV</address> <p>Pris 50,-</p> <p><b>BEM&AElig;RK!</b><br /> Dette køb er omfattet af forbrugeraftalelovens &sect;18, stk. 2, hvilket betyder, at du ikke har fortrydelsesret, når din bestilling er gennemført.</p><p>OBS! På grund af tekniske vanskeligheder, kan brugere med Hotmail e-mail adresser ikke benyttes af dette produkt. Benyt venligst en anden e-mail adresse.</p>
