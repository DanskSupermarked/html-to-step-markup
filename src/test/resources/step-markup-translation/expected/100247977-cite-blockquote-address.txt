Torsdag den 1. september inviterer <bold>føtex for de små</bold> til foredrag med den anerkendte sundhedsplejerske Helen Lyng Hansen i føtex Mariagervej Randers. Helen er blandt andet kendt fra hjemmesiden netsundhedsplejerske.dk og som forfatter til blandt andet bestselleren <underlined>Helens bog om børn og søvn – sådan får du dit barn til at sove</underlined>. Om børn og deres søvn siger Helen:<return/>
&gt; ”Kunsten er at forstå dit barns søvnmønster, udviklingstrin og behov for tryghed. Når du forstår det, ved du også, hvorfor det ikke hjælper bare at slukke lyset, lukke døren og gå.”<return/>
Helen giver dig en viden om børns søvn og udvikling, der gør det lettere for dig at forstå og tackle de søvnproblemer, I oplever derhjemme. Du får hjælp til at håndtere helt konkrete problemstillinger, og der er rig mulighed for at stille spørgsmål.<return/>
<return/>
Alle deltagere får udleveret en <bold>føtex for de små</bold> goodie bag med fine sager til både mor og den lille samt et gavekort til føtex på 50,- Der er begrænset antal pladser, så tøv ikke for længe med at bestille.<return/>
<return/>
Tidspunkt: Torsdag 1. september kl. 18-20<return/>
<return/>
føtex Mariagervej Randers<return/>
Mariagervej 175<return/>
8920 Randers NV<return/>
Pris 50,-<return/>
<return/>
<bold>BEMÆRK!</bold><return/>
Dette køb er omfattet af forbrugeraftalelovens §18, stk. 2, hvilket betyder, at du ikke har fortrydelsesret, når din bestilling er gennemført.<return/>
<return/>
OBS! På grund af tekniske vanskeligheder, kan brugere med Hotmail e-mail adresser ikke benyttes af dette produkt. Benyt venligst en anden e-mail adresse.
